examples/classification/4-tutorials/plot_main-tutorial-binary-classification.py

```diff
@@ -79,19 +78,14 @@
 n_classes = 2
 np.random.seed(42)
 X = np.vstack(
-    [
-        np.random.multivariate_normal(center, cov, n_samples)
-        for center, cov in zip(centers, covs)
-    ]
+    [np.random.multivariate_normal(center, cov, n_samples) for center, cov in zip(centers, covs)]
```

Comment:
I don't think this is very code reading friendly. Is this `black`?

Reply:
Yes, this is `black`

Reply:
Ugh, sometimes, black is really not friendly. 😆